commcare-support-library/build.gradle

```diff
@@ -1,26 +1,21 @@
 apply plugin: 'com.android.library'
-apply plugin: 'maven-publish'
+apply plugin: 'com.vanniktech.maven.publish'
 
 version '12.4'
 
-ext {
-    BINTRAY_USERNAME = project.properties['BINTRAY_USERNAME'] ?: ""
-    BINTRAY_PASSWORD = project.properties['BINTRAY_PASSWORD'] ?: ""
-}
-
 afterEvaluate {
     publishing {
         publications {
             release(MavenPublication) {
                 from components.release
-                groupId 'org.commcare'
+                groupId 'org.commcarehq.commcare'
                 artifactId 'support-library'
                 version this.version
 
                 pom {
                     name = 'CommCare Support Library'
                     description = 'Provides a set of utility functions for using CommCare Android mobile APIs'
-                    url = 'https://github.com/dimagi/commcare-support-library'
+                    url = 'https://github.com/dimagi/commcare-android/tree/master/commcare-support-library'
```

Comment:
This is the only part of the PR I'm not 100% sure on, since it's hard coupled to a branch.

This just provides a reference link for a reader, not anything automated, right?

Reply:
that's right.